debian/control

```diff
@@ -0,0 +1,15 @@
+Source: python-oq-libs
+Section: python
+Priority: extra
+Maintainer: Matteo Nastasi (GEM Foundation) <nastasi@openquake.org>
+Build-Depends: debhelper (>= 7.0.50~), lsb-release, curl, python (>=2.7), virtualenv, python-virtualenv, python-wheel
```

Comment:
Launchpad install deps with `--no-install-recommends``, so that's the reason why.